lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -54,7 +54,16 @@ def __init__(self, **kwargs):
             raise AnsibleError("Please pip install hvac to use this module")
 
         self.url = kwargs.get('url', ANSIBLE_HASHI_VAULT_ADDR)
-        self.token = kwargs.get('token')
+            
```

Comment:
CI failure due to PEP 8 issue:

```
2017-02-17 20:07:35 ERROR: PEP 8: lib/ansible/plugins/lookup/hashi_vault.py:57:1: W293 blank line contains whitespace (current)
```

You can run this test locally with `make pep8`.